src/disco/shred/fd_fec_resolver.c

```diff
@@ -402,6 +407,7 @@ int fd_fec_resolver_add_shred( fd_fec_resolver_t    * resolver,
       bmtrlist_push_head( bmtree_free_list, bmtree_mem );
       return FD_FEC_RESOLVER_SHRED_REJECTED;
     }
+    FD_MCNT_INC( SHRED, SHRED_REJECTED_INITIAL, (ulong)(-1L) ); /* Undo bad speculation */
```

Comment:
This is not valid, a counter cannot ever decrease in prometheus or the collector will assume that the application has restarted.

Reply:
Ah, I forgot the metrics tile could actually observe the count in between these. I was thinking it required metric write, but that's not true.